archive-locations.php

```diff
@@ -4,25 +4,101 @@
 <?php dt_print_breadcrumbs( null, __( "Locations" ) ); ?>
 
 <div id="content">
-
-    <div id="inner-content" class="row">
-
-        <main id="main" class="large-8 medium-8 columns" role="main">
-
-            <?php get_template_part( 'parts/content', 'locations-tabs.php' ); ?>
-
+    
+    <div id="inner-content" class="grid-x grid-margin-x">
+        
+        <main id="main" class="large-8 medium-8 cell" role="main">
+            
+            <?php
+            $args = array(
+                'post_type' => 'locations',
+            
+            );
+            $query1 = new WP_Query( $args );
```

Comment:
Following contacts and groups this code would be in the locations.php.
example `$locations = Disciple_Tools_Locations::get_locations();`
Note: the current get_locations does not support paging. For conversation: if we are expecting 100s or more locations we might need to update code in contacts-list